### PR TITLE
Fix transparent overlay for loading modal

### PR DESCRIPTION
**Pull request by jenniemc**
## Summary

This PR removes the `LoadingOverlay` wrapper to use `va-loading-indicator` web component directly on the "Choose a VA location" page to be consistent with the other VA appointment flow pages. Consulted with Design Team and Cierra approves the new loading message display on the "Choose a VA location" page.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#79894

## Testing done

Visual check at Choose a VA location page 

## Screenshots

Screenshot 2024-04-22 at 11 58 43 AM

## What areas of the site does it impact?

Loading message on choose a VA location page in the VA appointment flow and COVID appointment flow

## Acceptance criteria

- [ ] Loading message appears underneath the 'eyebrow' in the "Choose a VA location"
- [ ] Loading message does not use transparent overlay

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by vbahinwillit** [comment]
Question - I think the 'LoadingOverlay' component is the root cause of the problem. It wraps the 'va-loading-indicator' web component. The 'va-loading-indicator' is used directly in the rest of the codebase. Will it work if the 'LoadingOverlay' component is removed? It's only used in 2 modules.

**Review by powellkerry — DISMISSED** [review]

**Comment by jenniemc** [comment]
>Question - I think the 'LoadingOverlay' component is the root cause of the problem. It wraps the 'va-loading-indicator' web component. The 'va-loading-indicator' is used directly in the rest of the codebase. Will it work if the 'LoadingOverlay' component is removed? It's only used in 2 modules.

@vbahinwillit, that's a good thought. Let me look into the code and test it out.

**Comment by jenniemc** [comment]
@vbahinwillit Spoke to Cierra and showed her the idea of removing the transparent overlay. She liked it. Would you review this PR again? Thanks!

File: src/applications/vaos/sass/vaos.scss [hunk 1]
```diff
@@ -29,7 +27,8 @@ h1:focus {
 
 }
 .vaos__loading-overlay {
-  background: $vaos-loading-overlay;
+  background-color: var(--vads-color-white);
+  opacity: 0.9;
```
Comment (vbahinwillit) [inline src/applications/vaos/sass/vaos.scss:31]: I don't think this is needed any more. Only used in 'LoadingOverlay.jsx'.

**Review by vbahinwillit — APPROVED** [review]

File: src/applications/vaos/sass/vaos.scss (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/sass/vaos.scss:31]: I'll leave it there just in case we ever have to use the style

**Pull request merged**